src/open_inwoner/openzaak/zgw_imports.py

```diff
@@ -107,20 +107,16 @@ def import_zaaktype_configs() -> list[ZaakTypeConfig]:
         )
 
         for zaak_type in zaak_types:
-            catalog = None
-            if zaak_type.catalogus:
-                catalog = catalog_lookup.get(zaak_type.catalogus)
-                if not catalog:
-                    # weird edge-case: if the zaak_type has a catalogus-url but we don't have the object
-                    # TODO this is bad, log/raise something
-                    pass
+            try:
+                catalog = catalog_lookup[zaak_type.catalogus]
+            except KeyError:
+                raise ValueError(
```

Comment:
Is there a reason for raising `ValueError` instead of adding the msg to `KeyError`? According to the docs, `ValueError` is "Raised when an operation or function receives an argument that has the right type but an inappropriate value". Doesn't seem to be the case here.

Reply:
`RuntimeError` seems fine as well, though I would then `except KeyError as exc` and `raise RunTimeError from exc` so the stack trace is preserved. I wonder though if it's worth the effot to apply this distinction of perspectives to our case, since *we* are the caller (the calling code is even in the same package). I find this more intuitive in the case of a library.